doc/code/qml_qaoa.rst

```diff
@@ -1,6 +1,50 @@
 qml.qaoa
 ========
 
+This module provides a collection of methods that help in the construction of
+QAOA workflows.
+
+Here is an example of the PennyLane QAOA functionality being used to solve the
+`MaxCut <https://en.wikipedia.org/wiki/Maximum_cut>`__ problem:
+
+.. code-block:: python3
+
+    import pennylane as qml
+    from pennylane import qaoa
+    from networkx import Graph
+
+    # Defines the wires and the graph on which MaxCut is being performed
+
+    wires = range(3)
+    graph = Graph([(0, 1), (1, 2), (2, 0)])
+
+    # Defines the QAOA cost and mixer Hamiltonians
+
+    cost_h, mixer_h = qaoa.maxcut(graph)
+
+    # Defines a layer of the QAOA ansatz, from the cost and mixer Hamiltonians
+
+    def qaoa_layer(gamma, alpha):
+        qaoa.cost_layer(gamma, cost_h)
+        qaoa.mixer_layer(alpha, mixer_h)
+
+    # Repeatedly applies layers of the QAOA ansatz
+
+    def circuit(params, **kwargs):
```

Comment:
```suggestion
    def circuit(params):
```

Reply:
@co9olguy I just realized that we need to have the `**kwargs` argument in `circuit`, as `VQECost` requires the signature of the circuit input to be `(params, ***kwargs)`